### ISSUE TITLE
Visualising Residuals but they look weird

### ISSUE THREAD

**Issue by ShristiDasBiswas** [chaoyuaw/pytorch-coviar#95]
![Screenshot from 2023-10-31 00-00-06](https://github.com/chaoyuaw/pytorch-coviar/assets/99504345/dfbcd37c-07ef-4261-9b0f-d97b42464732)

Hi, my residuals look weird when i visualize them. What am I doing wrong? Any tips?

**Comment by Wendy-up** [comment]
Hello, It seems that you have been largely successful in reproducing the project, may I ask you did you pretrained the models on ImageNet or not, cause currently I got poor results compared with the original paper, I don't know if this maters or there is something else wrong so that I got the low accuracy.

really thanks!
